src/ray/raylet/scheduling_queue.h

```diff
@@ -40,6 +41,86 @@ enum class TaskState {
   WAITING_FOR_ACTOR,
 };
 
+class TaskQueue {
+ public:
+  /// \brief Append a task to queue.
+  ///
+  /// \param task_id The task ID for the task to append.
+  /// \param task The task to append to the queue.
+  /// \return Whether the append operation succeeds.
+  bool AppendTask(const TaskID &task_id, const Task &task);
+
+  /// \brief Remove a task from queue.
+  ///
+  /// \param task_id The task ID for the task to remove from the queue.
+  /// \return Whether the removal succeeds.
+  bool RemoveTask(const TaskID &task_id);
```

Comment:
Do we need both `RemoveTask` function signatures in the `TaskQueue` class?

Reply:
You are right we don't, I'm removing it.